### PR TITLE
removed unused large code comments

### PR DESCRIPTION
**Pull request by jaykay99992**
A large unused code comment of what seems to be like an old code, is removed for clarity.

### CODE REVIEW

File: jhotdraw-core/src/main/java/org/jhotdraw/draw/figure/LineConnectionFigure.java [hunk 1]
```diff
@@ -217,45 +217,6 @@ public void setStartConnector(Connector newStart) {
...
-  basicSetEndPoint(end);
-  }
-  }
-  }
-  changed();
-  }
-  }
-   */
-  // CLONING
-  // EVENT HANDLING
```
Comment (zchafik) [inline jhotdraw-core/src/main/java/org/jhotdraw/draw/figure/LineConnectionFigure.java:258]: the commented methods already implemented somewhere or definitely changing?

**Comment by jaykay99992** [comment]
@zchafik They are implemented only in that file, nowhere else. They are not being used at all but kept as a comment.

**Review by zchafik — COMMENTED** [review]
it's good the code will be more readable

**Comment by jaykay9999** [comment]
Sure, code merged

**Pull request merged**